src/main/java/org/jahia/modules/sam/healthcheck/probes/SupportedStackJVMProbe.java

```diff
@@ -0,0 +1,52 @@
+package org.jahia.modules.sam.healthcheck.probes;
+
+import org.graalvm.polyglot.Context;
+import org.jahia.bin.Jahia;
+import org.jahia.commons.Version;
+import org.jahia.modules.sam.Probe;
+import org.jahia.modules.sam.ProbeSeverity;
+import org.jahia.modules.sam.ProbeStatus;
+import org.jahia.modules.sam.core.ProbeStatusUtils;
+import org.jahia.osgi.BundleUtils;
+import org.osgi.framework.Bundle;
+import org.osgi.service.component.annotations.Component;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+
+@Component(service = Probe.class, immediate = true)
+public class SupportedStackJVMProbe implements Probe {
+
+    @Override
+    public ProbeStatus getStatus() {
+        String vmVendor = System.getProperty("java.vm.vendor", "Unknown");
+        Version jahiaVersion = new Version(Jahia.VERSION);
+        Version jvmVersion = new Version(System.getProperty("java.version", "Unknown"));
+
+        ProbeStatusUtils probStatusUtils = ProbeStatusUtils.getInstance();
```

Comment:
Same here, to be removed in favor of static method `ProbStatusUtils.aggregateStatus` usage.